app/services/recaptcha_enterprise_validator.rb

```diff
@@ -0,0 +1,56 @@
+class RecaptchaEnterpriseValidator < RecaptchaValidator
+  BASE_VERIFICATION_ENDPOINT = 'https://recaptchaenterprise.googleapis.com/v1/projects'.freeze
+
+  private
+
+  def assessment_url
+    UriService.add_params(
+      "#{BASE_VERIFICATION_ENDPOINT}/#{IdentityConfig.store.recaptcha_enterprise_project_id}" \
+        "/assessments",
```

Comment:
it's kind of a gross reuse of the API, but it does what we want...we could do this with `File.join` (`URI.join` doesn't work the sameO

```suggestion
      File.join(
        BASE_VERIFICATION_ENDPOINT,
        IdentityConfig.store.recaptcha_enterprise_project_id,
        '/assessments'
      ),
```

or if we switched string interpolation styles...

```suggestion
      format(
        '%{base_endpoint}/%{project_id}/assessments',
        base_endpoint: BASE_VERIFICATION_ENDPOINT,
        project_id: IdentityConfig.store.recaptcha_enterprise_project_id,
      ),
```

Reply:
Yeah, I think `format` could work quite nicely and help lend some clarity. Updated in 80f00242d.